### PR TITLE
Update analysing.md

### PR DESCRIPTION
**Pull request by floradanna**
Update analysing.md

### CODE REVIEW

File: pages/rdm_cycle/analysing.md [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
 
 ## What is data analysis?
 
-Data analysis encompasses all the different data manipulation and transformations that will help scientists to discover information or generate new knowledge.
-This is the step where the actual work on the data towards the goal of a research project takes place.
-The steps of the workflow in the Analysis phase of a project will often be repeated several times to explore the data as well as to optimize the process.
-According to the different types of data (quantitative or qualitative) the methods will differ.
-
-Data analysis follows the (often automated, batch) processing in the Processing stage.
+Data analysis consists in exploring the collected data to begin understanding the messages contained in a dataset and/or in applying mathematical formula (or models) to identify relationships between variables.
```
Comment (smza) [inline pages/rdm_cycle/analysing.md:9]: of exploring

File: pages/rdm_cycle/analysing.md [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+According to the different types of data (quantitative or qualitative) the data analysis methods will differ. Data analysis follows the (often automated, batch) data processing stage.
 
 ## Why is data analysis important?
 
-Since Data Analysis is the stage where new knowledge and information are generated, it can be considered as central in the research process.
-With many disciplines becoming data-oriented, more and more data intensive projects will occur and will involve experts from many thematic fields.
+Since data analysis is the stage where new knowledge and information are generated, it can be considered as central in the research process. Because of the relevance of the data analysis stage in research findings, it is essential that the analysis workflow applied to a dataset complies with the FAIR principles. Moreover, it is extremily important that the analysis workflow is reproducible by other researchers and scientists.  
+With many disciplines becoming data-oriented, more and more data intensive projects will occur and will require experts from many thematic fields.
```
Comment (smza) [inline pages/rdm_cycle/analysing.md:16]: more data intensive projects will arise which will require experts from dedicated fields

File: pages/rdm_cycle/analysing.md [hunk 3]
```diff
@@ -1,35 +1,31 @@
...
 
 ## What should be considered for data analysis?
+Because of the diversity of domains and techologies in Life Sciences, data can be either "small" or "big data". As a consequence, the methods and technical solutions used for data analysis might differ. The characteristics of "big data" are often summarized by a growing list of ["V's" properties: Volume, Velocity, Variety, Variability, Veracity, Visualization and Value](https://bigdatapath.wordpress.com/2019/11/13/understanding-the-7-vs-of-big-data/).
```
Comment (smza) [inline pages/rdm_cycle/analysing.md:19]: It should be "V's" or "V" properties

File: pages/rdm_cycle/analysing.md [hunk 4]
```diff
@@ -1,35 +1,31 @@
...
-* The variety of the data poses an integration challenge that can only adressed with the help of best practices that make data interoperable and reusable.
-* The Data Analysis phase relies on the previous steps (collection, processing) that will lay the foundations for the generation of new knowledge by providing acurate and trustworthy data.
-* For the analysis of data, you will first have to consider the computing environment and choose between several computing infrastructure types for e.g. cluster, cloud. You will also need to select the appropriate work environment according to your needs and expertise (command line, web portal).
-* The location of your data is important because of the needed need of proximity with computing resources. This can impact data transfer across the different infrastructures.
-* You will have to select the tools best suited for the analysis of your data. Resources such as [bio.tools](https://bio.tools) can be very helpful.
+* The data analysis stage relies on the previous stages (collection, processing) that will lay the foundations for the generation of new knowledge by providing acurate and trustworthy data.
```
Comment (smza) [inline pages/rdm_cycle/analysing.md:21]: accurate

File: pages/rdm_cycle/analysing.md [hunk 5]
```diff
@@ -1,35 +1,31 @@
...
+* The variety of the data poses an integration challenge that can only be adressed with the help of best practices that make data interoperable and reusable during the collection and/or process stage.
```
Comment (smza) [inline pages/rdm_cycle/analysing.md:22]: I'm not sure this should be a consideration for analysis stage

**Review by smza — COMMENTED** [review]
Suggested some style re-writing and a comment about misplaced consideration.

File: pages/rdm_cycle/analysing.md (hunk 5 above)
Comment (floradanna) [reply pages/rdm_cycle/analysing.md:22]: Maybe, but I don't think it harms to have it there.

Thanks for the suggestion, are you going to make these changes in a PR and merge it?

Comment (smza) [reply pages/rdm_cycle/analysing.md:22]: I'm inclined to delete it.

**Pull request merged**